gdx/src/com/badlogic/gdx/scenes/scene2d/ui/Label.java

```diff
@@ -76,7 +77,7 @@ public void setStyle (LabelStyle style) {
 		if (style == null) throw new IllegalArgumentException("style cannot be null.");
 		if (style.font == null) throw new IllegalArgumentException("Missing LabelStyle font.");
 		this.style = style;
-		cache = new BitmapFontCache(style.font, style.font.usesIntegerPositions());
+		cache = style.font.isDistanceField() ? new DistanceFieldBitmapFontCache(style.font, style.font.usesIntegerPositions()) : new BitmapFontCache(style.font, style.font.usesIntegerPositions());
```

Comment:
This isn't great. BitmapFont could have `newBitmapFontCache`, but I'm not sure I like that BitmapFont even knows about distance field font rendering. Maybe it would be better to just have a DistanceFieldFontCache (drop Bitmap from the name) that has the smoothing setting. Maybe Label could have something to set the BitmapFontCache, though so far that has been private.

Does anyone else have an opinion?

DistanceFieldFontCache breaks batching, which isn't so good.